relayer/src/error.rs

```diff
@@ -554,7 +555,76 @@ impl GrpcStatusSubdetail {
             .trim_start()
             .starts_with("account sequence mismatch")
     }
+
+    pub fn is_account_sequence_mismatch_that_can_be_ignored(&self) -> bool {
+        match parse_sequences_in_mismatch_error_message(self.status.message()) {
+            None => false,
+            Some((expected, got)) => expected < got,
+        }
+    }
+}
+
+fn parse_sequences_in_mismatch_error_message(message: &str) -> Option<(u64, u64)> {
+    let re = Regex::new(
+        r#"account sequence mismatch, expected (?P<expected>\d+), got (?P<got>\d+): incorrect account sequence"#
```

Comment:
We should have a comment to note that the regex may stop working if the error message is updated by the chain in the future.

Reply:
tried in https://github.com/informalsystems/ibc-rs/commit/132cd46276230574d3d0299b6192615cfe01cc78